osu.Game/Overlays/SkinEditor/SkinBlueprintContainer.cs

```diff
@@ -104,22 +104,24 @@ protected override void AddBlueprintFor(ISkinnableDrawable item)
 
         protected override bool OnKeyDown(KeyDownEvent e)
         {
+            bool holdingShift = e.ShiftPressed;
+
             switch (e.Key)
             {
                 case Key.Left:
-                    moveSelection(new Vector2(-1, 0));
+                    moveSelection(new Vector2(holdingShift ? -10 : -1, 0));
```

Comment:
Instead of having this ternary done in half a dozen places wouldn't it be better to have the shift handled in a method and then use that for each case

Reply:
Would something like this work?

```cs
int delta = e.ShiftPressed ? 10 : 1;

switch (e.Key)
{
    case Key.Left:
        moveSelection(new Vector2(-delta, 0));
        return true;
    case Key.Right:
        moveSelection(new Vector2(delta, 0));
        return true;
    case Key.Up:
        moveSelection(new Vector2(0, -delta));
        return true;
    case Key.Down:
        moveSelection(new Vector2(0, delta));
        return true;
}
```

Reply:
I'd probably put it after the switch but yeah that reads a lot better to me now.